src/server/metrics.rs

```diff
@@ -34,6 +34,19 @@ lazy_static! {
             &["type"]
         ).unwrap();
 
+    pub static ref GRPC_MSG_FAIL_COUNTER: CounterVec =
+        register_counter_vec!(
+            "tikv_grpc_msg_fail_total",
```

Comment:
It counts all received messages while not response successfully, including fail to send message to client.